apps/meteor/server/methods/sendWelcomeEmail.ts

```diff
@@ -0,0 +1,49 @@
+import { Users } from '@rocket.chat/models';
+import { Meteor } from 'meteor/meteor';
+
+import * as Mailer from '../../app/mailer/server/api';
+import { settings } from '../../app/settings/server';
+import { isSMTPConfigured } from '../../app/utils/server/functions/isSMTPConfigured';
+
+export async function sendWelcomeEmail(to: string): Promise<void> {
+	if (typeof to !== 'string') {
+		throw new Meteor.Error('error-invalid-arguments', 'Invalid arguments', {
+			method: 'sendWelcomeEmail',
+		});
+	}
+
+	if (!isSMTPConfigured()) {
+		throw new Meteor.Error('error-email-send-failed', 'SMTP is not configured', {
+			method: 'sendWelcomeEmail',
+		});
+	}
+
+	const email = to.trim();
+
+	const user = await Users.findOneByEmailAddress(email, { projection: { _id: 1 } });
+
+	if (!user) {
+		throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+			method: 'sendWelcomeEmail',
+		});
+	}
+
+	try {
+		let html = '';
+		Mailer.getTemplate('Accounts_UserAddedEmail_Email', (template) => {
+			html = template;
+		});
+
+		await Mailer.send({
+			to: email,
+			from: settings.get('From_Email'),
+			subject: settings.get('Accounts_UserAddedEmail_Subject'),
+			html,
+		});
```

Comment:
You're missing the `data` param in this object. This is used to replace user's `password`, `name` or `username` in the email content
My suggestion is to use the `_sendUserEmail` function [here](https://github.com/RocketChat/Rocket.Chat/blob/7396045d45e57861a076a3e25122cd0295d432b8/apps/meteor/app/lib/server/functions/saveUser.js#L42). Export it and use it here, you won't even need this try/catch

Reply:
Since this is an unforeseen problem in the flow of this feature I'll keep the current implementation and pass this information to product so they can decide.